src/terrain/terrain.hpp

```diff
@@ -589,6 +589,9 @@ class Terrain {
      */
     uint32_t get_voxel(int x, int y, int z) const;
 
+    inline int get_grass_grad_length() const{
+        return grass_grad_length;
```

Comment:
If this is private it should be underscored
If this is const it should be uppercase